### DISCUSSION TITLE
404 when opening notifications from gist subdomain

### DISCUSSION THREAD

**Discussion by pgrepds** [community/community#19938] [Projects and Issues] [answered]
Steps to reproduce:

1. Open https://gist.github.com/<Your UserName>.
2. Click on the notifications symbol in the top right corner next to the profile image.

Result:

It opens https://gist.github.com/notifications resulting in a "404 page not found".

Doing the same starting from the Github itself, e.g. from your profile page, works as expected. 

I guess that this has something to do with the subdomain name "gist".

**Comment by doguedogue** [comment]
you right man, I see the 404 page, do you have any gist created?

**Reply by pgrepds** [reply]
Yes several gists, both public and private ones.

**Comment by radar-blip** [comment, marked as answer]
Seen this before. Seems to be a small glitch or bug.
You can report it here:  https://support.github.com/contact/bug-report

**Reply by pgrepds** [reply]
Thanks, I was looking for this but I haven't found it. I have submitted an issue 👍🏻

**Reply by radar-blip** [reply]
Ok good. A couple members have reported it now so hopefully it will get fixed soon!
Have a great day!

**Reply by radar-blip** [reply]
Update: Seems to be fixed now.

**Reply by pgrepds** [reply]
Yes, this is fixed. The GitHub support just responded to the issue! 🚀
